src/components/FeaturedBanner/styles.css

```diff
@@ -1,14 +1,43 @@
-.donation-banner {
-  min-height: 39.375rem;
-}
 .donation-banner__icon {
-  width: 200px;
-  height: 200px;
+  width: 80px;
+  height: 80px;
   top: 0;
   left: 50%;
   transform: translate(-50%, -50%);
 }
+.donation-banner__btn {
+  padding: 1rem 1.5rem;
+}
 
 .donation-banner__btn:hover {
-  color: inherit !important;
+  color: rgb(37, 65, 81);
+}
+
+.donation-banner-desc {
+  font-size: 1.5rem;
+  line-height: 1.5;
+  margin-bottom: 2rem;
+}
+
+.donation-banner {
+  height: 2rem;
```

Comment:
2 rem only ?

Reply:
😞 . I set it to 25. by mistake 5 is omitted